nss-day-cohort-41/react-nutshell-the-macadamias
# Ajmac

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions

Start application, on login you should be redirected to the dashboard. in your application you can grab the id by going to window.sessionStorage.crednetials and using JSON.parse method.. on the object you have access to all of the current users information.

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
